src/plugins/kibana/public/management/sections/indices/upload/index.js

```diff
@@ -0,0 +1,15 @@
+import management from 'ui/management';
+import routes from 'ui/routes';
+import template from 'plugins/kibana/management/sections/indices/upload/index.html';
+import './directives/upload_wizard';
+
+routes.when('/management/data/upload/', {
+  template: template
+});
+
+management.getSection('data').register('upload', {
+  display: 'Upload CSV',
+  order: 10,
+  path: 'data/upload/'
```

Comment:
What are your thoughts on making the route more specific, like _csv_? I am wondering if upload might be too generic. What if we allow uploading of data directories, snapshots, etc?